### ISSUE TITLE
🐛 [ml_vision] iOS example doesn't work

### ISSUE THREAD

**Issue by jaumard** [FirebaseExtended/flutterfire#4841]
## Bug report

**Describe the bug**
iOS example for ml_vision plugin is not working at all.

### Steps to reproduce

1. Open the example project
2. run it on iOS device
3. Click MaterialBarcodeScanner
4:
```
7.3.0 - [Firebase/MLKit][I-MLK006007] Can't load a barcode engine class, please check that MLVisionBarcodeModel is installed.
7.3.0 - [Firebase/MLKit][I-MLK006006] Barcode engine is nil.
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: PlatformException(Error 13, com.firebase.ml, Barcode engine is nil., null)
```

### Expected behavior

Should be able to read barcode. It should contain `pod 'Firebase/MLVisionBarcodeModel'` but once you add it you're blocked by https://github.com/FirebaseExtended/flutterfire/issues/4625

---

## Additional context

podFile is not setup with the right information to have it working 

---

**Comment by Kleak** [comment]
@jaumard and I are maintaining the https://github.com/rushio-consulting/flutter_camera_ml_vision and I was looking for updating everything, on android no problem but we are blocked for iOS due to this.

Also flutter_camera_ml_vision is used by our respective company so it would be great to be able to update everything :)

**Comment by markusaksli-nc** [comment]
Hi @jaumard 
I'm going to close this as a duplicate of https://github.com/FirebaseExtended/flutterfire/issues/4625 since it will be blocked by that anyway and the error is to be expected without the ML pod.
Could you comment this info there?
Thank you

**Comment by jaumard** [comment]
doesn't make sens at all... it's not a duplicate, it's just block by  #4625 so that one should have the priority over this one, but once  #4625 is done, example will still not work as they are now...

**Comment by markusaksli-nc** [comment]
A fix to that issue would also require a fix to the example, it just needs to get mentioned there.
